src/features/offer/components/OfferBody/OfferBody.native.test.tsx

```diff
@@ -110,6 +114,8 @@ describe('<OfferBody />', () => {
           />
         )
 
+        await screen.findByText('Envoyer sur \n Instagram')
```

Comment:
Pour info, sans line break ça marche aussi :
```suggestion
        await screen.findByText('Envoyer sur Instagram')
```